app/Http/Controllers/API/v1/OutboundController.php

```diff
@@ -17,4 +19,23 @@ public function sendPing()
     {
         return WmsJabar::sendPing();
     }
+
+    /**
+     * getNotification function for API {{base_url}}/api/v1/poslog-notify
+     * Get notification from POSLOG to read update logistic request data
+     *
+     * @return void
+     */
+    public function getNotification(Request $request)
+    {
+        $param = [
+            'request_id' => 'required'
+        ];
+        $response = Validation::validate($request, $param);
+        if ($response->getStatusCode() === 200) {
+            $getOutboundById = WmsJabar::getOutboundById($request);
+            $response = response()->format(200, 'notification accepted', $getOutboundById);
```

Comment:
Kang @rindibudiaramdhan saya lupa, di Laravel itu kyknya ada enum/constant buat response, jadi nggak perlu hardcode HTTP response code. Misal utk HTTP 200 ada constant Resonse::SUCCESS

CMIIW cc kang @ayocodingit

Reply:
> use Illuminate\Http\Response;
> 
> Response::HTTP_OK

Reply:
siap akang2kuh otewe

Reply:
baris 38 nya gak sekalian kang @rindibudiaramdhan

Reply:
Iya betul kang @rindibudiaramdhan , boleh sekalian ganti `200` yang ada di baris 37 :grin: 

```suggestion
            $response = response()->format(Response::HTTP_OK, 'notification accepted', $getOutboundById);
```

Reply:
@rindibudiaramdhan selain ini aku udah ok yah.

Reply:
untuk code yang lain bisa dicek disini ya kaka2 https://gist.github.com/jeffochoa/a162fc4381d69a2d862dafa61cda0798

Reply:
Mantap 😁